apps/desktop/src/lib/vbranches/virtualBranch.ts

```diff
@@ -96,10 +96,24 @@ export class VirtualBranchService {
 	}
 
 	private async listVirtualBranches(): Promise<VirtualBranch[]> {
-		return plainToInstance(
-			VirtualBranches,
-			await invoke<any>('list_virtual_branches', { projectId: this.projectId })
-		).branches;
+		const response = await invoke<any>('list_virtual_branches', { projectId: this.projectId });
+		const virtualBranches = plainToInstance(VirtualBranches, response);
+
+		if (virtualBranches.dependencyCalculationErrors.length > 0) {
+			this.handleDependencyCalculationErrors(virtualBranches.dependencyCalculationErrors);
+		}
+
+		return virtualBranches.branches;
+	}
+
+	private handleDependencyCalculationErrors(errors: DependencyCalculationError[]) {
+		for (const e of errors) {
+			console.error(`DependencyCalculationError:
+${e.errorMessage}
+Stack: ${e.stackId}
+Commit: ${e.commitId}
+Path: ${e.path}`);
+		}
```

Comment:
Print the errors to the dev console.
Don't display toasts because these errors are potentially raised repeatedly.